### PR TITLE
Feature: Swedish Personal Identity number date

### PR DESCRIPTION
**Pull request by crkt**
**Type of change**

- [x] New feature
- [ ] Bug fix
- [ ] Security patch
- [ ] Documentation update

**Description**

I have been using this api in other projects and found that as a consumer I end up doing things like this:

```typescript
const pnr = Personnummer.parse(personalIdentityNumber);

const { fullYear: year, month, day: dayString } = pnr;

let day = parseInt(dayString, 10);
if (pnr.isCoordinationNumber()) {
  day = day - 60;
}

const birthDate = new Date(`${year}-${month}-${day}`);
```

With the goal of getting a date and then working with the date to get diffs in different units, such as months, days or years. With `getAge` that already exists I lose some granularity such as months. I went ahead and made a suggestion for this using the code in `getAge` to avoid breaking things.

**Related issue**
None

**Motivation**

It would be great to able to have a `Date` to work with to integrate nicely with other libs such as `date-fns` and getting `differenceInMonths` or other units. And also avoiding having to know about coordination numbers in my consuming code. In my example I have to check if the personal identity number is a coordination number to figure out the date of birth.

**Checklist**

- [x] I have read the **CONTRIBUTING** document.
- [x] I have read and accepted the **Code of conduct**
- [x] Tests passes.
- [ ] Style lints passes.
- [ ] Documentation of new public methods exists.
- [x] New tests added which covers the added code.
- [ ] Documentation is updated.
- [ ] This PR includes breaking changes.

### CODE REVIEW

**Comment by frozzare** [comment]
Hi, 

Thanks for the pull request, I agree that it's a good function to add so you can work with the date. But before we can merge this we need to update the specification and the reset of the maintainers need give their input, can you open a issue and describe the need for the function in https://github.com/personnummer/meta? 

The specification exists so all implementation of Personnummer acts the same way and we have 19 packages to update with this change so it's important that we have this in the specification and a new version before we just merge and adds different things to different packages.

**Comment by crkt** [comment]
Sure, I'll open an issue in the meta package. Thanks!

**Comment by frozzare** [comment]
We've merge the [new specification](https://github.com/personnummer/meta#get-date), so if you like you can update you're pull request with the new function name `getDate` and we can merge it. I'll do the rest of the stuff needed for the new specification and after that we can probably release a new version 🙌

**Comment by crkt** [comment]
I've updated the name and removed mentions of "dateOfBirth" in the code. If there is anything else you need from me just let me know.

**Review by frozzare — APPROVED** [review]

**Comment by frozzare** [comment]
@crkt it's released now :)

**Pull request merged**
